### PR TITLE
Make mdldec output is more readable and conformant to crowbar/studiomdl.

### PR DESCRIPTION
**Pull request by nekonomicon**

### CODE REVIEW

File: utils/mdldec/qc.c [hunk 1]
```diff
@@ -197,43 +198,67 @@ static void WriteTextureRenderMode( FILE *fp )
 {
 	int		  i;
 	mstudiotexture_t *texture;
+	qboolean	  addnewline = false;
```
Comment (a1batross) [inline utils/mdldec/qc.c:201]: What if you use ftell here? Let's say:

```
long pos = ftell( fp );
... 
if( ftell != pos )
   fputs( fp, "\n" );
```

File: utils/mdldec/qc.c [hunk 2]
```diff
@@ -296,7 +320,7 @@ static void WriteAttachmentInfo( FILE *fp )
 	if( !model_hdr->numattachments )
 		return;
 
-	fprintf( fp, "\n// %i attachment(s)\n", model_hdr->numattachments );
+	fprintf( fp, "// %i attachment(s)\n", model_hdr->numattachments );
```
Comment (a1batross) [inline utils/mdldec/qc.c:324]: `fprintf( fp, "// %s attachment%s\n", model_hdr->numattachments, model_hdr->numattachments > 1 ? "s" : "" );`

:)

File: utils/mdldec/qc.c [hunk 3]
```diff
@@ -373,7 +401,7 @@ static void WriteControllerInfo( FILE *fp )
 	if( !model_hdr->numbonecontrollers )
 		return;
 
-	fprintf( fp, "\n// %i bone controller(s)\n", model_hdr->numbonecontrollers );
+	fprintf( fp, "// %i bone controller(s)\n", model_hdr->numbonecontrollers );
```
Comment (a1batross) [inline utils/mdldec/qc.c:405]: `fprintf( fp, "// %s bone controller%s\n", model_hdr->numbonecontrollers, model_hdr->numbonecontrollers > 1 ? "s" : "" );`

:)

File: utils/mdldec/qc.c [hunk 4]
```diff
@@ -402,7 +432,7 @@ static void WriteHitBoxInfo( FILE *fp )
 	if( !model_hdr->numhitboxes )
 		return;
 
-	fprintf( fp, "\n// %i hit box(es)\n", model_hdr->numhitboxes );
+	fprintf( fp, "// %i hit box(es)\n", model_hdr->numhitboxes );
```
Comment (a1batross) [inline utils/mdldec/qc.c:443]: `fprintf( fp, "// %s hit box%s\n", model_hdr->numhitboxes, model_hdr->numhitboxes > 1 ? "es" : "" );`

File: utils/mdldec/qc.c [hunk 5]
```diff
@@ -430,10 +462,10 @@ static void WriteSequenceInfo( FILE *fp )
 	mstudioseqdesc_t	*seqdesc;
 
 	if( model_hdr->numseqgroups > 1 )
-		fputs( "\n$sequencegroupsize 64\n", fp );
+		fputs( "$sequencegroupsize 64\n\n", fp );
 
 	if( model_hdr->numseq > 0 )
-		fprintf( fp, "\n// %i animation sequence(s)\n", model_hdr->numseq );
+		fprintf( fp, "// %i animation sequence(s)\n", model_hdr->numseq );
```
Comment (a1batross) [inline utils/mdldec/qc.c:499]: `fprintf( fp, "// %s animation sequence%s\n", model_hdr->numseq, model_hdr->numseq > 1 ? "s" : "" );`

File: utils/mdldec/qc.c (hunk 1 above)
Comment (nekonomicon) [reply utils/mdldec/qc.c:201]: for what?

File: utils/mdldec/qc.c [hunk 6]
```diff
@@ -371,13 +371,13 @@ static void WriteBodyGroupInfo( FILE *fp )
 
 			if( !Q_strncmp( model->name, "blank", 5 ) )
```
Comment (a1batross) [inline utils/mdldec/qc.c:373]: Is it should catch everything that starts with `blank`?

Comment (nekonomicon) [reply utils/mdldec/qc.c:373]: Yep.

File: utils/mdldec/qc.c (hunk 1 above)
Comment (a1batross) [reply utils/mdldec/qc.c:201]: So you don't have to add `addnewline = true` to every branch here.

Comment (nekonomicon) [reply utils/mdldec/qc.c:201]: how?

Comment (a1batross) [reply utils/mdldec/qc.c:201]: How what? You already added it.

Comment (nekonomicon) [reply utils/mdldec/qc.c:201]: Oh... I understood...

Comment (a1batross) [reply utils/mdldec/qc.c:201]: Isn't it easier? %)))

**Comment by a1batross** [comment]
is it ready now?

**Comment by nekonomicon** [comment]
Yep.

**Pull request merged**